changelog.d/2897.added.md

```diff
@@ -0,0 +1 @@
+Add link to My Stuff that opens a pop up with a generated QR Code linking to that site
```

Comment:
```suggestion
Add link to My Stuff that opens a pop up with a generated QR Code linking to that page
```

Reply:
Huh, I thought I fixed that...guess I forgot to commit it